Firestore/README.md

```diff
@@ -23,35 +23,30 @@ command script import ~/path/to/firebase-ios-sdk/scripts/lldb/firestore.py
 
 ### Running Integration Tests
 
+By default, run integration tests against the Firestore Emulator. This is much
```

Comment:
Consider rewording this sentence to begin with "Prefer running the integration tests..."

I find that by saying "By default" this suggests that it is a default setting that you get unless you specify otherwise. But if I understand correctly, the purpose of this sentence is to encourage people to use the emulator in lieu of production.

Reply:
Done!

(Note that it kind of is a default setting--out of the box the the source tree is configured for running against the emulator.)